### ISSUE TITLE
keras.util.to_categorial throws error

### ISSUE THREAD

**Issue by gilad207** [Oreobird/tf-keras-deep-head-pose#6]
__array__() takes 1 positional argument but 2 were given

can  fix it pls?

**Comment by nhathoang0110** [comment]
i have this error too. can I help me ?

**Comment by gilad207** [comment]
i just didnt use this code with the error

**Comment by nhathoang0110** [comment]
I don't understand your answer. 
can you provide suitable tensorflow and keras version?

**Comment by gilad207** [comment]
i am not the maintainer of this repo. i used this repo in my project and i had to delete the buggy code.

**Comment by nhathoang0110** [comment]
can you take me your source code that you edit ? thanks you very much
